src/player.js

```diff
@@ -116,6 +111,11 @@ class Player extends FakeEventTarget {
     return {};
   }
 
+  _initializeMembers(): void {
```

Comment:
I think in general this is not required.
The only place we initialize or reset our general members is in the constructor and in the destroy, and config and tracks can be set to default on initialization to these values.
In this case I prefer inlining.